reana-ui/src/pages/workflowDetails/components/LauncherLabel.js

```diff
@@ -0,0 +1,46 @@
+/*
+  This file is part of REANA.
+  Copyright (C) 2022 CERN.
+
+  REANA is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
+*/
+
+import PropTypes from "prop-types";
+import { Label } from "semantic-ui-react";
+
+export default function LauncherLabel({ url }) {
+  /**
+   * Return appropriate Label component props according to the given launcher URL.
+   * @param {String} url Submission workflow launcher URL.
+   * @returns {Object} Appropriate component props.
+   */
+  function getProps(url) {
+    const externalSrcProps = {
+      as: "a",
+      href: url,
+      target: "_blank",
+      rel: "noopener noreferrer",
+    };
+
+    if (!url) {
+      return { content: "reana-client", icon: "terminal" };
+    } else if (url.startsWith("https://zenodo.org/")) {
+      return { content: "Zenodo", color: "blue", ...externalSrcProps };
```

Comment:
BTW why Zenodo blue and not the others? Were you thinking of starting to differentiate between sources in that way? Zenodo one colour, GitHub another, GitLab another, CERN Open Data another, and URL another? 

(I guess people may be wondering why colours are different and whether it has some functional meaning or just aesthetic ones... An alternative would be to use the same grayish colour and differentiate between the sites just by leading icons. )

![b4](https://user-images.githubusercontent.com/517546/157293898-878d0134-1fd5-43fb-80a4-4eb237d4cc05.png)

Reply:
It was an attempt to personalize somehow each label. The other labels have icons, Zenodo doesn't, that's the only reason why. I see that I failed miserably. I'll leave it gray as the others.